float32[]       path            # the trajectory of the shape
uint32          shapeID         # the ID of the instance of the shape
string          shapeType       # the type of the shape
uint32          shapeType_code  # the code of the type of the shape
uint8[]         paramsToVary    # which parameters are varying in the shape model
float32[]       paramValues     # the parameter values used to make this shape
